tests/src/test/scala/com/nvidia/spark/rapids/SparkQueryCompareTestSuite.scala

```diff
@@ -201,8 +201,6 @@ trait SparkQueryCompareTestSuite extends AnyFunSuite with BeforeAndAfterAll {
   def withCpuSparkSession[U](f: SparkSession => U, conf: SparkConf = new SparkConf()): U = {
     val c = conf.clone()
       .set(RapidsConf.SQL_ENABLED.key, "false") // Just to be sure
-      // temp work around to unsupported timestamp type
-      .set("spark.sql.parquet.outputTimestampType", "TIMESTAMP_MICROS")
```

Comment:
Fixes:  cuDF writes timestamp as int96, while Spark write timestamp as int64:

Reply:
Looks like I added that in 3 years ago and somehow the GPU version got cleaned up but the CPU version didn't.  Thanks for fixing this.